vcs/src/main/resources/ext.py

```diff
@@ -52,6 +56,15 @@ def writeln(s):
     write(s)
     sys.stdout.write(encode('\n'))
 
+def _match_exact(root, cwd, files, badfn=None):
+    """
+    Wrapper for mercurial.match.exact that ignores some arguments based on the used version
+    """
+    if mercurial.util.versiontuple() < 5:
```

Comment:
Two questions here:
1. How does this check work? `versiontuple()` returns a tuple, e.g. `(4, 9, 2)`, how can you compare a tuple to an `int`?
2. `mercurial.util.versiontuple()` is fairly new (added 2015) whereas `mercurial.util.version()` has been around since 2009. Is it possible to implement the check using `mercurial.util.version()` instead? Like `mercurial.util.version().startswith("5")`?

Reply:
1. My bad, supposed to compare the first element, which is not quite what happens when comparing with an `int` (though it works in the specific case I tested).
2. Yeah, I was afraid of that. Will switch to using `version()`. (could also go with the `try ... except` approach if you prefer)

Reply:
1. No worries, that is why we review :+1:
2. Personally I would prefer something like `mercurial.util.version().startswith("5")` _if_ that approach works, I didn't check too carefully what `mercurial.util.version()` returns

Reply:
I did (and then I noticed this shiny `versiontuple`, it just returns a string like `5.0.2`

Reply:
Ok, great!